src/cli.py

```diff
@@ -39,18 +42,32 @@ def cli():
                 # Can be deleted for flight
                 print("calling __setPIPE")
                 gs.__setPIPE__()
+                data = bytearray(libcsp.packet_get_data(toSend))
+
+            if (
+                server == sysVals.APP_DICT.get('OBC') and 
```

Comment:
I believe this was mentioned in the previous PR but the server prefixes are now 'EX2', 'ARI', and 'YUK'